renovate.json

```diff
@@ -3,5 +3,15 @@
   "extends": [
     "config:base"
   ],
-  "schedule": ["every weekend"]
+  "schedule": ["every weekend"],
+  "packageRules": [
+    {
+      "matchPackagePrefixes": ["org.immutables"],
```

Comment:
2.9.2 has bom, so we can use it instead.
However, sigstore-java is not compatible with 2.9.2